### PR TITLE
added my name

### PR DESCRIPTION
**Pull request by misraelson**
🤘

### CODE REVIEW

File: mjfox-was-here.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
-[
+[ Matthew,
```
Comment (Qard) [inline mjfox-was-here.json]: can you put the name in the empty space line?

**Comment by misraelson** [comment]
DONE ✔️

File: mjfox-was-here.json (hunk 1 above)
Comment (misraelson) [reply mjfox-was-here.json]: Done

**Comment by ab218** [comment]
💯

**Comment by chowdhurian** [comment]
@misraelson As you gave them the option when you open the PR, maintainers of the upstream are able to make changes to commit to branches you've created from forks.

@qard Please take another look. The conflicts should be resolved now!

**Pull request merged**